package.json

```
@@ -137,36 +127,31 @@
     "web components"
   ],
   "scripts": {
-    "build": "npm-run-all build:sass sass:render dist:js build:api test build:docs bundler postinstall",
+    "build": "npm-run-all build:sass sass:render dist:js build:api test build:docs cssLint bundler postinstall ",
     "build:ci": "npm-run-all sweep build",
-    "build:api": "wca analyze 'src/auro-combobox.js' --outFiles docs/api.md",
-    "build:demo": "npm-run-all build demo:rm:build demo:new:build demo:copy:index demo:copy:demo demo:update:index",
-    "build:dev:assets": "npm-run-all build:sass:demo build:sass:component postCss:component sass:render",
+    "build:api": "wca analyze 'src/auro-badge.js' --outFiles docs/api.md",
```

```suggestion
    "build:api": "wca analyze 'src/auro-combobox.js' --outFiles docs/api.md",
```

Please stop copy and pasting this into each repo.